src/js/rollup.config.js

```diff
@@ -12,7 +14,10 @@ function config({ input, format, minify, ext = "js" }) {
       format,
       sourcemap: true,
     },
+    external: ["path", "fs/promises", "node-fetch", "vm"],
     plugins: [
+      commonjs(),
+      nodeResolve(),
```

Comment:
Could you please add a comment on what this does? I'm still not sure after reading their readme.